docs/concepts/storage/volumes.md

```diff
@@ -141,22 +141,37 @@ spec:
 ### hostPath
 
 A `hostPath` volume mounts a file or directory from the host node's filesystem
-into your pod.  This is not something that most Pods will need, but it offers a
+into your pod. This is not something that most Pods will need, but it offers a
 powerful escape hatch for some applications.
 
 For example, some uses for a `hostPath` are:
 
 * running a container that needs access to Docker internals; use a `hostPath`
   of `/var/lib/docker`
 * running cAdvisor in a container; use a `hostPath` of `/dev/cgroups`
+* allowing a pod to specify whether a given hostPath should exist prior to the
+  pod running, whether it should be created, and what it should exist as
+
+Supported HostPath Types are:
+
+| Value | Behavior |
+|:------|:---------|
+| *unset* | For backwards compatible, leave it empty if unset. **Default value**. |
```

Comment:
change line 159 to this?

| `` | Empty string (default) or unset are for backward compatibility. It means that Kubernetes performs *no* checks before mounting the hostPath volume. |